Assets/Scripts/Possession/Throwable.cs

```diff
@@ -10,10 +11,12 @@ public class Throwable : MonoBehaviour, IPossessable
     private Vector3 _releasePosition;
 
     [Header("Display Controls")]
-    [SerializeField] [Range(10, 100)] private int _linePoints = 25;
-    [SerializeField] [Range(0.01f, 0.25f)] private float _timeBetweenPoints = 0.1f;
+    [SerializeField] [Range(10, 100)] private int _linePoints;
+    [SerializeField] [Range(0.01f, 0.25f)] private float _timeBetweenPoints;

```

Comment:
Ik zou de default waarde van timebetween points erin laten omdat het zo'n vreemde waarde is. Deze waarde erin laten geeft iemand anders een goed begin punt om mee te werken.